### PR TITLE
Update RELEASE_NOTES.md

### PR DESCRIPTION
**Pull request by rfrost-hpe**
# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by jsl-hpe — APPROVED** [review]
I think I'm ethically obliged to reject my manager's first PR, but I'm going to claim I misclicked.

**Review by heemstra — APPROVED** [review]

**Comment by heemstra** [comment]
I'm just here to witness the historical event.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,10 +6,19 @@ documentation improvements. This page lists some of the highlights.
...
+* Fixed dns has forgotten about all Compute/Application nodes - CAST-34613
+* Fixed  worker rebuild workflow: update-bss-no-wipe - CAST-33450
+* Fixed unbound not publishing to prometheus - CAST-31867
+* Fixed spire server slow to respond and crashing -CAST-33608
+* Fixed Add worker nodes to running system - CAST-34633
+* Fixed missing entries in haproxy on storage nodes - CAST-33719
+* Fixed Unable to to monitor the progress of a boot of single node in the power on process when using BOS V2 - CAST-32362
+* Fixed cray-powerdns-manager not adding compute nodes to external DNS - CAST-31486
+* Fixed unbound slamming our dns server with requests - CAST-31712
+* Fixed prerequisites.sh makes DNS nonfunctional for some minutes. - CAST-32978
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md]:
```suggestion
* Resolved CASTS:
    * `CAST-31486`: `cray-powerdns-manager` not adding compute nodes to external DNS
    * `CAST-31712`: Unbound slamming DNS server with requests
    * `CAST-31867`: Fixed Unbound not publishing to Prometheus
    * `CAST-32362`: Unable to to monitor the progress of a boot of single node in the power on process when using BOS V2
    * `CAST-32978`: `prerequisites.sh` makes DNS nonfunctional for some minutes
    * `CAST-33450`: Worker rebuild workflow: `update-bss-no-wipe`
    * `CAST-33608`: `spire` server slow to respond and crashing
    * `CAST-33719`: Missing entries in `haproxy` on storage nodes
    * `CAST-33922`: `PXE-E18: Server response timeout`
    * `CAST-34613`: DNS has forgotten about all Compute/Application nodes
    * `CAST-34633`: Add worker nodes to running system
    * `CAST-34692`: Kubernetes DNS not working/will not start
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
-CASTS resolved
-CAST-32797 SMA rsyslog-collector pod file descriptor leak and resource exhaustion
-CAST-32143  CSM 1.3 unbound is losing connection to upstream DNS
 
+* Resolved CASTS:
+    * `CAST-31486`: `cray-powerdns-manager` not adding compute nodes to external DNS
+    * `CAST-31712`: Unbound slamming DNS server with requests
+    * `CAST-31867`: Fixed Unbound not publishing to Prometheus
+    * `CAST-32362`: Unable to to monitor the progress of a boot of single node in the power on process when using BOS V2
+    * `CAST-32978`: `prerequisites.sh` makes DNS nonfunctional for some minutes
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md]:
```suggestion
    * `CAST-32978`: `prerequisites.sh` makes DNS not functional for some minutes
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
 ### New Software Support
-CASTS resolved
-CAST-32797 SMA rsyslog-collector pod file descriptor leak and resource exhaustion
-CAST-32143  CSM 1.3 unbound is losing connection to upstream DNS
 
+* Resolved CASTS:
+    * `CAST-31486`: `cray-powerdns-manager` not adding compute nodes to external DNS
+    * `CAST-31712`: Unbound slamming DNS server with requests
+    * `CAST-31867`: Fixed Unbound not publishing to Prometheus
+    * `CAST-32362`: Unable to to monitor the progress of a boot of single node in the power on process when using BOS V2
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:14]: This is kind of misleading. Reading this, I would assume that BOS v2 had been changed in some way to allow this. However, I believe we actually just added some documentation to help customers collect the desired information. And that documentation probably went into CSM 1.4 as well, so it's not really part of CSM 1.5, per se.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-32978`: `prerequisites.sh` makes DNS not functional for some minutes
+    * `CAST-33450`: Worker rebuild workflow: `update-bss-no-wipe`
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:16]: This description does not do a good job of conveying what was fixed or changed, IMO. Also, this was fixed in CSM 1.4.1

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-33608`: `spire` server slow to respond and crashing
+    * `CAST-33719`: Missing entries in `haproxy` on storage nodes
+    * `CAST-33922`: `PXE-E18: Server response timeout`
+    * `CAST-34613`: DNS has forgotten about all Compute/Application nodes
+    * `CAST-34633`: Add worker nodes to running system
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:21]: This description is pretty vague in terms of what the problem was that was fixed. Also, it is fixed in CSM 1.3 and CSM 1.4

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
 ## New
 
 ### New Software Support
-CASTS resolved
-CAST-32797 SMA rsyslog-collector pod file descriptor leak and resource exhaustion
-CAST-32143  CSM 1.3 unbound is losing connection to upstream DNS
 
+* Resolved CASTS:
+    * `CAST-31486`: `cray-powerdns-manager` not adding compute nodes to external DNS
+    * `CAST-31712`: Unbound slamming DNS server with requests
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:12]: This was also fixed in CSM 1.4.2, so it's not exactly a new fix in CSM 1.5.0. If we're listing everything that was fixed in the previous CSM patch releases, this list could be a lot longer. :)

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-31867`: Fixed Unbound not publishing to Prometheus
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:13]: This went into CSM 1.4.0, per the CAST.

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-32362`: Unable to to monitor the progress of a boot of single node in the power on process when using BOS V2
+    * `CAST-32978`: `prerequisites.sh` makes DNS not functional for some minutes
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:15]: This was also fixed in CSM 1.3.4 and CSM 1.4.1.

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-33450`: Worker rebuild workflow: `update-bss-no-wipe`
+    * `CAST-33608`: `spire` server slow to respond and crashing
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:17]: This was fixed in CSM 1.4.3

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-33719`: Missing entries in `haproxy` on storage nodes
+    * `CAST-33922`: `PXE-E18: Server response timeout`
+    * `CAST-34613`: DNS has forgotten about all Compute/Application nodes
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:20]: This was fixed in CSM 1.3.4 and CSM 1.4.1

File: RELEASE_NOTES.md [hunk 11]
```diff
@@ -6,10 +6,20 @@ documentation improvements. This page lists some of the highlights.
...
+    * `CAST-34633`: Add worker nodes to running system
+    * `CAST-34692`: Kubernetes DNS not working/will not start
```
Comment (mharding-hpe) [inline RELEASE_NOTES.md:22]: This was fixed in CSM 1.4.4

File: RELEASE_NOTES.md (hunk 3 above)
Comment (rfrost-hpe) [reply RELEASE_NOTES.md:14]: removed it

File: RELEASE_NOTES.md (hunk 5 above)
Comment (rfrost-hpe) [reply RELEASE_NOTES.md:21]: removed it

File: RELEASE_NOTES.md (hunk 7 above)
Comment (rfrost-hpe) [reply RELEASE_NOTES.md:13]: removed it

File: RELEASE_NOTES.md (hunk 9 above)
Comment (rfrost-hpe) [reply RELEASE_NOTES.md:17]: removed it

**Pull request merged**